app/views/projects/dataset/keys/_key.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= f.fields_for 'values_attributes[]', Dataset::Value.new do |nested_form| %>
```

Comment:
The format of the params will be compatible with `accepts_nested_attributes_for :values` on Dataset::ValueSet